lib/dradis/plugins/settings.rb

```diff
@@ -44,6 +44,15 @@ def is_default?(key, value)
       value.to_s == @default_options[key.to_sym].to_s
     end
 
+    def engine_enable
+      enable = db_setting(:enable)
+      enable ? (enable == '1') : nil
+    end
+
+    def toggle_engine(value)
```

Comment:
It seems better to implement an `enable` and `disable` method instead to make it more consistent with the app. See https://github.com/secroots/dradis-pro/blob/develop/app/models/user.rb#L133